CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# Citizen and Contributor Code of Conduct
+
+## 1. Purpose
+A primary goal of NV Access and NVDA is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, nationality, socioeconomic status, education, level of experience and religion (or lack thereof).
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behaviour.
+We invite all those who participate in the NVDA community to help us create safe and positive experiences for everyone. NVDA is a user driven initiative. 
+
+## 2. Open Source Citizenship
+A supplemental goal of this Code of Conduct is to increase open source citizenship by encouraging participants to recognise and strengthen the relationships between our actions and their effects on our community. 
+Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+## 3. Expected Behaviour
+The following behaviours are expected and requested of all community members:
+* To participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+* To exercise consideration and respect in your speech and actions.
+* To attempt collaboration before conflict.
+* To provide helpful contributions to an issue in order to encourage its progression

```

Comment:
```suggestion
* To provide helpful contributions to an issue to encourage its progression

```

Reply:
Thanks for the feedback @codeofdusk .
As the document itself is essentially a policy set forth by NV Access Limited, changes to the actual text itself cannot be addressed here. This pr is merely to add the document (as is) to the NVDA repository, and link to it from various locations.
The document text conforms to accepted Australian English spelling rules (which I'll admit is perhaps a little loose). E.g. My Australian English spell checker does allow a z in prioritize and an s in license. However, NV Access may choose to review this text in the future, and at that stage will update the code of conduct file in the repository.